Makefile

```diff
@@ -60,10 +60,12 @@ build-slim:
 test: run-unit-tests
 
 .PHONY: lint
+lint: diffnoprefix := $(shell sh -c "git config --get --show-origin diff.noprefix | awk '/file:\.git\/config\s+\S+/ {print \$$2}'")
 lint: build-slim build-test
-	# Note: Make sure that you run `git config diff.noprefix false` in this repo
-	# See this issue for more details: https://github.com/golangci/golangci-lint/issues/948
+	# To workaround: https://github.com/golangci/golangci-lint/issues/948
```

Comment:
Either "To work around" or "A workaround for" :-)

Reply:
Fixed, thanks.